cloud

```diff
@@ -637,6 +653,7 @@ function cloud_functions_withdraw() {
     cloud_function_withdraw "$project" "${prefix}send_notification"
     cloud_function_withdraw "$project" "${prefix}spool_notifications"
     cloud_function_withdraw "$project" "${prefix}load_queue"
+    cloud_function_withdraw "$project" "${prefix}pick_notifications"
```

Comment:
Same here, could you move the function to before `send_notifications`? Here the order is not critical, I think, but it's better to preserve code uniformity in general.